packages/server/tests/acceptance.spec.ts

```diff
@@ -374,13 +383,23 @@ describe('RPC Server Acceptance Tests', async function () {
     });
 
     it('should execute "eth_getBalance" for newly created account with 5000 HBAR', async function () {
-        const { accountInfo } = await utils.createEthCompatibleAccount(client, null, 5000);
+        const accountResponse = await utils.createEthCompatibleAccount(client, null, 5000);
```

Comment:
Can we reduce the 5000 everywhere to like 10 hbar.
I want to make it cheaper to run when pointing to a remote deployment

Reply:
If you want to address this in PR 211 that works for me